pytorch/inference/docker/1.10/py3/sdk1.19.0/Dockerfile.neuron

```diff
@@ -77,11 +76,10 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     typing \
     # Below 2 are included in miniconda base, but not mamba so need to install
     conda-content-trust \
-    charset-normalizer \
- && /opt/conda/bin/conda clean -ya
+    charset-normalizer
 
 RUN conda install -c conda-forge \
-    opencv>=4.6.0 \
+    opencv \
```

Comment:
Why remove the constraint here?

Reply:
The constraint does nothing to update the version of opencv-python.

Reply:
We need the opencv version to be above a certain minimum limit in order to prevent conda from auto-resolving to find an opencv version with a security vulnerability in it.

Reply:
Synced offline - Removing the constraint is a fix to a known issue where the version constraint does not actually have an impact on the version installed.

Resolving this conversation.